_topic_map.yml

```diff
@@ -684,6 +684,9 @@ Topics:
 - Name: Disabling Features using Feature Gates
   File: disabling_features
   Distros: openshift-origin,openshift-enterprise
+- Name: Using Statefulsets
+  File: assembly_stateful-sets
```

Comment:
@mburke5678 - hey, the file naming conventions are changing this week. We are not using assembly in the name of the files anymore.